GetVisibleServices/__tests__/handler.test.ts

```diff
@@ -85,7 +85,8 @@ describe("GetVisibleServicesHandler", () => {
       })
     };
     const getVisibleServicesHandler = GetVisibleServicesHandler(
-      blobStorageMock as any
+      blobStorageMock as any,
+      false
```

Comment:
Can we add a test when this param is `true`?

Reply:
Unit test added with ea72764